docs/source/markdown/podman-run.1.md

```diff
@@ -831,8 +831,8 @@ Run container in systemd mode. The default is **true**.
 
 The value *always* enforces the systemd mode is enforced without
 looking at the executable name.  Otherwise, if set to **true** and the
-command you are running inside the container is systemd, _/usr/sbin/init_
-or _/sbin/init_.
+command you are running inside the container is systemd, _/usr/sbin/init_,
+_/sbin/init_ or _/usr/local/sbin/init_.
```

Comment:
I know this was pre-existing, but why do we have the differing formatting for the same message?  I'd vote to italicize both (add the `_` 's) to create's sentence.

Reply:
I was puzzled about the different formatting, too. Eventually I decided to keep the mark-up consistent.